demo/src/Components/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const Tooltip = ({ isOpen, id, handleClick }) => (
+  <div className="o-grid__item u-1/4@sm">
+    <div className={classNames('c-tooltip', { 'is-open': isOpen })}>
+      <button className="c-tooltip__toggle" onClick={() => { handleClick(id) }}>toggle me: {id}</button>
```

Comment:
don't need any kind of toggle button as we'll be using `:hover` and `:focus` for tooltips.

Reply:
done